config/initializers/good_job.rb

```diff
@@ -10,4 +10,18 @@
   config.good_job.poll_interval = 5
   # config.good_job.enable_cron = true
   # config.good_job.cron = { example: { cron: '0 * * * *', class: 'ExampleJob'  } }
-end
\ No newline at end of file
+end
+
+# baseline of 0, higher is sooner
+ApplicationJob.priority = 0
+AttachFilesToWorkJob.priority = -1
+Bulkrax::ImportWorkJob.priority = -5
+Bulkrax::ImporterJob.priority = -20
+Bulkrax::CreateRelationshipsJob.priority = -10
+CharacterizeJob.priority = 30
+ContentDepositEventJob.priority = -50
+ContentUpdateEventJob.priority = -50
+CreateDerivativesJob.priority = 40
+Bulkrax::ScheduleRelationshipsJob.priority = 50
+Hyrax::GrantEditToMembersJob.priority = 10
+IngestJob.priority = 10
```

Comment:
```suggestion
ApplicationJob.priority = 0
AttachFilesToWorkJob.priority = -1
Bulkrax::CreateRelationshipsJob.priority = -10
Bulkrax::ImporterJob.priority = -20
Bulkrax::ImportWorkJob.priority = -5
Bulkrax::ScheduleRelationshipsJob.priority = 50
CharacterizeJob.priority = 30
ContentDepositEventJob.priority = -50
ContentUpdateEventJob.priority = -50
CreateDerivativesJob.priority = 40
Hyrax::GrantEditToMembersJob.priority = 10
IngestJob.priority = 10
```

Reply:
```suggestion
Bulkrax::ScheduleRelationshipsJob.priority = 50
CreateDerivativesJob.priority = 40
CharacterizeJob.priority = 30
Hyrax::GrantEditToMembersJob.priority = 10
IngestJob.priority = 10
ApplicationJob.priority = 0
AttachFilesToWorkJob.priority = -1
Bulkrax::ImportWorkJob.priority = -5
Bulkrax::CreateRelationshipsJob.priority = -10
Bulkrax::ImporterJob.priority = -20
ContentDepositEventJob.priority = -50
ContentUpdateEventJob.priority = -50
```

Reply:
the first suggestion is to list these alphabetically. the second, is listing in order of priority. I'm fine with either.

Reply:
great idea! thanks :)